### PR TITLE
fix 5 small tasks

### PR DESCRIPTION
**Pull request by Kasvit**
**STATUS**
READY

**DESCRIPTION**
Tasks:
http://prntscr.com/lhz0el and same for tasks 
wrong date http://prntscr.com/lhz310
input for comment http://prntscr.com/lhz3ep
http://prntscr.com/lhz5l1 link to workspace in header <Workspace> / Project name
align labels to the left side http://prntscr.com/lhza6r

**CHECK LIST**
- [ ] specs added
- [ ] specs passed
- [ ] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs

### CODE REVIEW

**Comment by Kasvit** [comment]
image
image
image
image
image
image

**Comment by ShevchukTania** [comment]
approved

File: app/views/account/comments/_form.html.slim [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .container.box
   = simple_form_for [:account, commentable, comment], remote: true do |f|
     = f.input :body, placeholder: 'Add a comment...', label: 'New comment',
-              wrapper: :textarea, required: true
+              wrapper: :textarea, required: true, as: :text
```
Comment (romaslmd) [inline app/views/account/comments/_form.html.slim]: pls follow the pattern:
`f.input <attribute>, <as: [input]>, <*options>, <*html_options>`

File: app/views/account/profiles/_form.html.slim [hunk 2]
```diff
@@ -1,9 +1,9 @@
 = simple_form_for @user, url: [:account, :profile] do |f|
   = f.input :avatar, label: "Select your profile avatar", wrapper: :upload_file
-  = f.input :first_name, wrapper: :bulma_input, required: true
-  = f.input :last_name, wrapper: :bulma_input, required: true
-  = f.input :job_role, wrapper: :bulma_input
-  = f.input :department, wrapper: :bulma_input
-  = f.input :about, wrapper: :textarea
+  = f.input :first_name, wrapper: :bulma_input_label_left, required: true
```
Comment (igorkasyanchuk) [inline app/views/account/profiles/_form.html.slim]: this is not good
I don't want to have 1000 inputs

it's possible to specify class using simple_form

remove this wrapper and do it

File: config/initializers/simple_form.rb [hunk 3]
```diff
@@ -18,6 +18,15 @@
     b.use :error, wrap_with: { tag: :div, class: "help is-danger" }
   end
 
+  config.wrappers :bulma_input_label_left, class: "field", error_class: :field_with_errors do |b|
```
Comment (igorkasyanchuk) [inline config/initializers/simple_form.rb]: remove

File: config/initializers/simple_form.rb [hunk 4]
```diff
@@ -38,6 +47,15 @@
     b.use :error, wrap_with: { tag: :div, class: "help is-danger" }
   end
 
+  config.wrappers :textarea_label_left, class: "field", error_class: :field_with_errors do |b|
```
Comment (igorkasyanchuk) [inline config/initializers/simple_form.rb]: remove

File: app/views/account/profiles/_form.html.slim (hunk 2 above)
Comment (Kasvit) [reply app/views/account/profiles/_form.html.slim]: = f.input :first_name, class: "has-text-left", wrapper: :bulma_input, required: true
 Its doesn`t work. We need to add class inside the wrapper
image
And without any wrapper:
image

**Comment by ShevchukTania** [comment]
пул реквест проглчнула

**Comment by supersem** [comment]
переглянув

**Review by igorkasyanchuk — APPROVED** [review]

**Pull request merged**